smart_campus/app/templates/app/station_edit.html

```diff
@@ -0,0 +1,154 @@
+{% extends 'app/base.html' %}
+{% load static %}
+
+{% block stylesheet %}
+  {{ block.super }}
+  <link rel="stylesheet" type="text/css" href="{% static "app/css/station_edit.css" %}" />
+{% endblock %}
+{% block content %}
+<div class="container content">
+  <!-- Display Error -->
+  {% for field in form %} 
+    {% for error in field.errors %}
+        <li class='text-danger'><strong>{{ field.label }}: {{ error|escape }}</strong></li>
```

Comment:
Unify the style of using `{{}}`.
i.e. Do not mix use `{{ var }}` and `{{var}}`

Reply:
okay, 
checked in all templates